packages/maskbook/src/plugins/PluginUI.ts

```diff
@@ -9,7 +9,9 @@ export const PluginUI: ReadonlySet<PluginConfig> = plugins
 import { Flags } from '../utils/flags'
 import { sideEffect } from '../utils/side-effects'
 import { DHedgePluginDefine } from './dHEDGE/define'
+import { GoodGhostingPluginDefine } from './GoodGhosting/define'
 
 sideEffect.then(() => {
     if (Flags.dhedge_enabled) plugins.add(DHedgePluginDefine)
+    if (Flags.good_ghosting_enabled) plugins.add(GoodGhostingPluginDefine)
```

Comment:
Hello, @OsamaElHariri thanks for your contribution. Please use the new way to define a plugin, you can see many examples in the plugins folder. `dhedge` is the only last plugin not migrated to the new definition standard and we should not add a new definition here.

Reply:
FYI, all new APIs are well documented in packages\plugin-infra\src\types.ts and the new plugin structure can be seen in packages\plugins\example.

Reply:
I went over the example of how the updated plugin infra looks like and updated the PR. This plugin is now defined using the updated way of handling plugins